docs/api/session.md

```diff
@@ -42,6 +42,21 @@ To create a `Session` with `options`, you have to ensure the `Session` with the
 `partition` has never been used before. There is no way to change the `options`
 of an existing `Session` object.
 
+### `session.fromPath(path[, options])`
+
+* `path` string
+* `options` Object (optional)
+  * `cache` boolean - Whether to enable cache.
+
+Returns `Session` - A session instance from the absolute path as specified by the `path`
+string. When there is an existing `Session` with the same absolute path, it
+will be returned; otherwise a new `Session` instance will be created with `options`. The
+call will throw an error if the path is not an absolute path or an empty string.
```

Comment:
I would interpret this as "you have to pass in either an absolute path or an empty string, otherwise it throws"
but actually you always need to pass in an absolute path because empty strings are also invalid with the current implementation

Reply:
Thanks @ajafff. I have  updated the documentation to avoid ambiguity.

Thanks,
George.